README.md

```diff
@@ -32,3 +32,47 @@ or run only tests using
 ```bash
 vendor/bin/phing tests
 ```
+
+### Running development version
+
+The executable is bin/phpstan
+Rut it as
+bin/phpstan analyze -c your-config.neon -l 8 path-to-analyze
```

Comment:
Should use code fences (like other similar commands in this file)

Reply:
Done.

Reply:
The development version should be run as `vendor/bin/phing phpstan` instead.

Reply:
`vendor/bin/phing phpstan` is to run `phpstan` on itself. I mean the command to analyse custom code using dev phpstan as one modifies phpstan.